src/domain/usecase/create-spot.ts

```diff
@@ -22,14 +23,24 @@ export type CreateSpot = (options: CreateSpotOptions) => Promise<Spot>
 export const createSpot = ({
   spotRepository,
   authenticationService,
-}: Dependencies): CreateSpot => ({ name, description, location, authorId }) => {
+}: Dependencies): CreateSpot => ({
+  name,
+  description,
+  location,
+  authorId,
+  tags,
+}) => {
   authenticationService.throwIfNotLoggedIn()
 
+  if (tags.length === 0) {
+    throw new Error('At least 1 tag is required')
+  }
+
   validateLatitude(location.latitude)
   validateLongitude(location.longitude)
 
   const id = uuid.v4()
-  const spot = new Spot(id, name, description, location, authorId)
+  const spot = new Spot(id, name, description, location, authorId, tags)
```

Comment:
This fails the [max-params](https://eslint.org/docs/rules/max-params) lint rule. We should convert our use case constructors to objects IMO.